frontend/packages/console-shared/src/components/dashboard/dashboard-card/DashboardCardLink.tsx

```diff
@@ -67,6 +70,8 @@ type DashboardCardPopupLinkProps = {
   onShow?: () => void;
   onHide?: () => void;
   position?: PopoverPosition;
+  maxWidth?: string;
+  hasAutoWidth?: boolean;
```

Comment:
Why this  ?

Reply:
https://github.com/openshift/console/pull/9426#discussion_r675529322